assets/model_monitoring/components/src/model_monitor_metric_outputter/run.py

```diff
@@ -24,16 +27,20 @@ def run():
     """Output metrics."""
     # Parse arguments
     arg_parser = argparse.ArgumentParser()
+    arg_parser.add_argument("--monitor_name", type=str)
     arg_parser.add_argument("--signal_name", type=str)
     arg_parser.add_argument("--signal_type", type=str)
     arg_parser.add_argument("--signal_metrics", type=str)
+    arg_parser.add_argument("--metric_timestamp", type=str)
     arg_parser.add_argument("--signal_output", type=str)
 
     args = arg_parser.parse_args()
 
     metrics: List[Row] = read_mltable_in_spark(args.signal_metrics).collect()
 
-    metrics_dict = MetricOutputBuilder(metrics).get_metrics_dict()
+    runmetric_client = RunMetricClient()
+    metrics_dict = MetricOutputBuilder(runmetric_client, args.monitor_name, args.signal_name, metrics) \
```

Comment:
nit: the trailing backslash is not needed when concatenating a dot.

Reply:
https://msdata.visualstudio.com/Vienna/_workitems/edit/2659503
Added this WIT for fixing this comment